packages/instrumenter/src/mutators/arithmetic-operator-mutator.ts

```diff
@@ -28,7 +28,17 @@ export class ArithmeticOperatorMutator implements NodeMutator {
     return [];
   }
 
-  private isSupported(operator: string): operator is keyof typeof ArithmeticOperators {
-    return Object.keys(this.operators).includes(operator);
+  private isSupported(operator: string, node: types.BinaryExpression): operator is keyof typeof ArithmeticOperators {
+    if (!Object.keys(this.operators).includes(operator)) {
+      return false;
+    }
+
+    const stringTypes = ['StringLiteral', 'TemplateLiteral'];
```

Comment:
On further thought, this should probably be a whitelist not a blacklist. It's a bit weird, but something like:

```
const log = (thingToLog) => console.log(thingToLog);
const concat = 1 + log;
console.log(concat)
```

Will result in `"1(thingToLog) => console.log(thingToLog)"`. What do you think @nicojs, should we do a whitelist of things that actually make sense for an ArithmeticOperator, or keep it simple and only make exceptions for strings and template strings?

Reply:
I think a whitelist can become big and hard(er) to maintain. I personally think this is fine to start with. It's better than what we have now.